packages/context/src/inject.ts

```diff
@@ -45,7 +46,7 @@ export interface InjectionMetadata {
    */
   decorator?: string;
   /**
-   * Control if the dependency is optional, default to false
+   * Control if the dependency is optional. Default to `false`.
```

Comment:
I find both the old and the new versions difficult to parse. How about the following?

```suggestion
   * Control if the dependency is optional. Default value: `false`.
```